tc/sgx/trusted_worker_manager/enclave/CMakeLists.txt

```diff
@@ -127,7 +127,7 @@ ENDFUNCTION()
 
 # XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-FILE(GLOB PROJECT_HEADERS *.h ${TCF_CORE_DIR}/common/*.h)
+FILE(GLOB PROJECT_HEADERS *.h)
```

Comment:
These changes to Cmake file is no more needed as the latest code already have it covered by PR - https://github.com/hyperledger/avalon/pull/235